### PR TITLE
Use RabbitMQ for quick install guide

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

This PR:

- Revises the five-minute install guide to use RabbitMQ as the transport instead of Redis
- Splits out the Enterprise guide from the Core guide
- Removes jokes (Sorry, no fun allowed)

Tested with Vagrant and CentOS 7

To do: Copy to all versions post-review

## Motivation and Context

Promoting best practices and creating a better experience for Enterprise users

### CODE REVIEW

**Review by majormoses — APPROVED** [review]
Core content looks good, no idea on enterprise.

**Review by cwjohnston — CHANGES_REQUESTED** [review]
In https://github.com/sensu/sensu-docs/pull/714/ we've updated our repo definitions to specify `gpgcheck=1` and provide a value for `gpgkey` attribute. 

I think we should do likewise in the repo definitions we set up here.

File: content/sensu-enterprise/3.1/quick-start/five-minute-install.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+using the `curl` utility (and piping the result to the [`jq` utility][10]):
+
+{{< highlight shell >}}
+sudo yum install jq curl -y
+curl -s http://127.0.0.1:4567/clients | jq .{{< /highlight >}}
+
+The Sensu Enterprise API should return a JSON array of Sensu clients similar to this:
+
+{{< highlight shell >}}
+$ curl -s http://127.0.0.1:4567/clients | jq .
```
Comment (rgeniesse) [inline content/sensu-enterprise/3.1/quick-start/five-minute-install.md]: This command is repeated from line 231. Maybe remove one? Not sure which makes the most sense.

File: content/sensu-core/1.5/quick-start/five-minute-install.md [hunk 2]
```diff
@@ -148,52 +174,29 @@ Sensu Core users:
...
 
 {{< highlight shell >}}
 sudo yum install jq curl -y
 curl -s http://127.0.0.1:4567/clients | jq .{{< /highlight >}}
 
-If the Sensu API returns a JSON array of Sensu clients similar to this:
+The Sensu API should return a JSON array of Sensu clients similar to this:
 
 {{< highlight shell >}}
 $ curl -s http://127.0.0.1:4567/clients | jq .
```
Comment (rgeniesse) [inline content/sensu-core/1.5/quick-start/five-minute-install.md]: Unrelated to these changes, but this command is repeated from line 194. Maybe remove one? Not sure which makes the most sense.

**Comment by rgeniesse** [comment]
FWIW, I ran through all the commands in the Core and Enterprise steps. Both created working Sensu stacks. I left a comment about a repeated API command. Other than that, 💯 . Awesome work! New users should have a better experience installing Sensu.

**Comment by apaskulin** [comment]
- Fixed @cwjohnston's request in https://github.com/sensu/sensu-docs/pull/718/commits/de09b77af28cfaf00ac54a7e0f296e439cd008fa
- Fixed @rgeniesse's request in https://github.com/sensu/sensu-docs/pull/718/commits/ea209c9f45b693d6006db0dfbd5d5f039351a01f
- Removed the server configuration step per @cwjohnston's suggestion and updated API response in https://github.com/sensu/sensu-docs/pull/718/commits/25251db7986bdf81bfe2edd3f1f164fd74cb7d4a

**Review by rgeniesse — APPROVED** [review]
💯 💯

File: content/sensu-core/0.29/quick-start/five-minute-install.md [hunk 3]
```diff
@@ -88,50 +80,70 @@ protected-mode no{{< /highlight >}}
...
 echo '{
- "transport": {
-   "name": "redis"
- },
- "api": {
-   "host": "127.0.0.1",
-   "port": 4567
- }
-}' | sudo tee /etc/sensu/config.json{{< /highlight >}}
+  "rabbitmq": {
```
Comment (cwjohnston) [inline content/sensu-core/0.29/quick-start/five-minute-install.md:100]: Sensu's RabbitMQ defaults are user `guest` password `guest` and vhost `/`. The guest user is only allowed to connect from localhost. If we want to advise folks to use non-default credentials and/or vhost, we will also need to provide the relevant rabbitmqctl commands for adding user, setting password, creating vhost and assigning permissions on that vhost to the user.

Comment (apaskulin) [reply content/sensu-core/0.29/quick-start/five-minute-install.md:100]: Fixed in https://github.com/sensu/sensu-docs/pull/718/commits/2a3900e41c38f3fc190bc9e2b54ed59d52b630dc

**Review by cwjohnston — APPROVED** [review]
Looks good! :shipit:

**Pull request merged**